packages/core/src/AvatarGroup/stories/AvatarGroup.stories.tsx

```diff
@@ -41,10 +41,10 @@ export const Main: StoryObj<HvAvatarGroupProps> = {
         <HvAvatar alt="Ben" src="https://i.imgur.com/56Eeg1g.png" />
         <HvAvatar alt="Beatrice" src="https://i.imgur.com/bE7vg3N.png" />
         <HvAvatar alt="Wayne" src="https://i.imgur.com/ea22egF.png" />
-        <HvAvatar alt="Clara Soul" src="https://i.imgur.com/6sYhSb6.png" />
-        <HvAvatar alt="Ben" src="https://i.imgur.com/56Eeg1g.png" />
-        <HvAvatar alt="Beatrice" src="https://i.imgur.com/bE7vg3N.png" />
-        <HvAvatar alt="Wayne" src="https://i.imgur.com/ea22egF.png" />
+        <HvAvatar alt="Clara" src="https://i.imgur.com/6sYhSb6.png" />
+        <HvAvatar alt="Steve" src="https://i.imgur.com/kshWwMd.jpeg" />
```

Comment:
This link (5th avatar) doesn't seem to work as expected:

<img width="1025" alt="Screenshot 2024-04-08 at 11 33 30" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/8da9038c-4b65-4c91-b316-c74f196ec53e">